src/systems/velocity_control/VelocityControl.cc

```diff
@@ -86,15 +135,53 @@ void VelocityControl::Configure(const Entity &_entity,
     return;
   }
 
-  // Subscribe to commands
-  std::string topic{"/model/" + this->dataPtr->model.Name(_ecm) + "/cmd_vel"};
+  // Subscribe to model commands
+  std::string modelTopic{"/model/" + this->dataPtr->model.Name(_ecm) + "/cmd_vel"};
   if (_sdf->HasElement("topic"))
-    topic = _sdf->Get<std::string>("topic");
+    modelTopic = _sdf->Get<std::string>("topic");
   this->dataPtr->node.Subscribe(
-    topic, &VelocityControlPrivate::OnCmdVel, this->dataPtr.get());
-
-  ignmsg << "VelocityControl subscribing to twist messages on [" << topic << "]"
+    modelTopic, &VelocityControlPrivate::OnCmdVel, this->dataPtr.get());
+  ignmsg << "VelocityControl subscribing to twist messages on ["
+         << modelTopic << "]"
          << std::endl;
+
+  // Ugly, but needed because the sdf::Element::GetElement is not a const
+  // function and _sdf is a const shared pointer to a const sdf::Element.
+  auto ptr = const_cast<sdf::Element *>(_sdf.get());
+
+  if (!ptr->HasElement("link1_name") || !ptr->HasElement("link2_name"))
```

Comment:
About the comment I had before for accepting duplicate `link_name` elements. I was thinking of something like:
```
  sdf::ElementPtr sdfElem = ptr->GetElement("link_name");
  while (sdfElem)
  {
    this->dataPtr->linkNames.push_back(sdfElem->Get<std::string>())
    sdfElem = sdfElem->GetNextElement("link_name");
  }
```
This lets users specify a list link names instead of limiting them to 2. So instead of

```
<plugin ...>
  <link_name1>my_link_name_1</link_name1>
  <link_name2>my_link_name_2</link_name2>
</plugin>
```
users can do:

```
<plugin ...>
  <link_name>my_link_name_1</link_name>
  <link_name>my_link_name_2</link_name>
  <link_name>my_link_name_3</link_name>
</plugin>
```

For creating topics, we can do it in a loop

```
  for (const auto &linkName : this->dataPtr->linkNames)
  { 
    std::string linkTopic{"/model/" + this->dataPtr->model.Name(_ecm) +
                          "/link/" + linkName + "/cmd_vel"};
    this->dataPtr->node.Subscribe(
      linkTopic, &VelocityControlPrivate::OnLinkCmdVel, this->dataPtr.get());
      ignmsg << "VelocityControl subscribing to twist messages on ["
             << linkTopic << "]"
             << std::endl;
  }

```

In `OnLinkCmdVel`, we can just store the velocities in a map instead of individual variables. We also won't need different functions for updating the velocity, i.e. `UpdateUpperLinkVelocity` and `UpdateLowerLinkVelocity`. We can just have one generic function to update velocities for all links